runtime/j9vm31/CMakeLists.txt

```diff
@@ -58,14 +58,28 @@ set(compile_flags_to_remove
 	"\"-Wa,SYSPARM(BIT64)\""
 	"-DJ9ZOS39064"
 	"-qxplink=noback"
+	"-m64"
 )
 
+# Flags to be appended while compiling
+set(compile_flags_to_append
+	"-m32n"
+	"-mzos-float-kind=ieee"
+	"-Wno-c++11-narrowing"
+)	
```

Comment:
Please fix the line-ending issues (like this line).

Reply:
Should be removed with latest commit.